### PR TITLE
Wait until cilium-cni binary is ready when ENABLE_CILIUM_PLUGIN is set

### PR DESCRIPTION
**Pull request by jingyuanliang**
I'm avoiding bash-specific syntax ([[ ]] instead of [ ]) due to #119 (even if I'm fixing the configuration file to use bash instead of sh, I'm not sure about the direction of the next step, so better to be safe here).

### CODE REVIEW

File: scripts/install-cni.sh [hunk 1]
```diff
@@ -56,23 +56,32 @@ fi
...
+  cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin#,{\"type\": \"bandwidth\",\"capabilities\": {\"bandwidth\": true}}#g")
 else
-	cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin##g")
+  cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin##g")
 fi
 
 if [ "${ENABLE_CILIUM_PLUGIN}" == "true" ]
 then
-        cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniCiliumPlugin#,{\"type\": \"cilium-cni\"}#g")
+  echo "Adding Cilium plug-in:"
```
Comment (MrHohn) [inline scripts/install-cni.sh:73]: nit: "Adding Cilium plug-in to the CNI config." to be clear?

**Review by MrHohn — APPROVED** [review]
Thanks Jingyuan, LGTM with a minor comment.
/approve

File: scripts/install-cni.sh (hunk 1 above)
Comment (jingyuanliang) [reply scripts/install-cni.sh:73]: Done

**Review by MrHohn — APPROVED** [review]
/lgtm

**Pull request merged**